Docs  image and tabs extended paredown (#1139)
carbon-design-system/carbon-for-ibm-dotcom-website: src/pages/components/images.mdx

</Column>
</Row>

> <Caption>When an image has a caption, the user can hover over the image and click to trigger a lightbox overlay with more information.</Caption>

Reviewer: ```suggestion
<Caption>Users can hover over the image and click to open a lightbox overlay that includes a larger image and more information.</Caption>
```
